Lib/fontTools/ttLib/tables/_k_e_r_n.py

```diff
@@ -17,7 +19,7 @@ class table__k_e_r_n(DefaultTable.DefaultTable):
 
 	def getkern(self, format):
```

Comment:
The name of this function is so wrong!  Shall we remove it or what?

Reply:
I doubt anyone ever used it because it was throwing AttributeError because of missing "version" attribute on anything other than format-0 subtables (the only one implemented). Sure, I can remove it. We can add a better one when we implement other subtable formats.